pkg/client/actionclient.go

```diff
@@ -83,9 +141,15 @@ func (hcg *actionClientGetter) ActionClientFor(obj client.Object) (ActionInterfa
 	}
 	postRenderer := DefaultPostRendererFunc(rm, actionConfig.KubeClient, obj)
 	return &actionClient{
-		conf:               actionConfig,
-		defaultInstallOpts: []InstallOption{WithInstallPostRenderer(postRenderer)},
-		defaultUpgradeOpts: []UpgradeOption{WithUpgradePostRenderer(postRenderer)},
+		conf: actionConfig,
+
+		defaultGetOpts:       hcg.defaultGetOpts,
+		defaultInstallOpts:   append([]InstallOption{WithInstallPostRenderer(postRenderer)}, hcg.defaultInstallOpts...),
+		defaultUpgradeOpts:   append([]UpgradeOption{WithUpgradePostRenderer(postRenderer)}, hcg.defaultUpgradeOpts...),
```

Comment:
Just a nit, but if these fields are by default initialized to an empty slice it might look a tad cleaner to do something like:
```suggestion
		defaultInstallOpts:   append(hcg.defaultInstallOpts, WithInstallPostRenderer(postRenderer)),
		defaultUpgradeOpts:   append(hcg.defaultUpgradeOpts, WithUpgradePostRenderer(postRenderer)),
```

A very nitpicky change, but it prevents us from having to initialize a slice with the `...PostRenderer()` call.

Reply:
I did this on purpose so that someone could specify their own `PostRenderer` that would take precedent over the one we set (the functional options are executed in order, so later ones get a chance to overwrite what earlier ones have done).

I'll add a comment to explain this though since it isn't obvious.